docs/specification/README.md

```diff
@@ -59,10 +59,15 @@ The following records are allowed to appear in the data section:
 - [Chunk](#chunk-op0x06)
 - [Message Index](#message-index-op0x07)
 - [Metadata](#metadata-op0x0C)
+
 - [Data End](#data-end-op0x0F)
 
 The last record in the data section MUST be the [Data End](#data-end-op0x0F) record.
 
+#### Use of chunk records
+
+MCAP files can have Schema, Channel, and Message records written directly to the data section, or they can be written into Chunk records to facilitate indexing and compression. Schema, Channel, and Message records must either all be written inside chunks, or all outside of chunks. A data section containing Chunk records may not contain any Schema, Channel, or Message records and vice-versa.
```

Comment:
> A data section containing Chunk records may not contain any Schema, Channel, or Message records and vice-versa.

They're still in the data section, just inside a chunk. I wonder if there's another way to word this.

Reply:
Reworded, might be slightly better?